src/components/ColorPicker/CustomColorPicker.tsx

```diff
@@ -55,10 +57,11 @@ export const CustomColorPicker: FC<Omit<ColorPickerProps, "palette">> = ({
             <div className="tw-flex tw-gap-2 tw-w-full tw-h-[200px]">
                 <div className="tw-relative tw-flex-grow tw-overflow-hidden tw-rounded">
                     <Saturation
-                        hsl={hsl}
-                        hsv={hsv}
+                        rgb={parsedColor.toRgb()}
+                        hsl={parsedColor.toHsl()}
+                        hsv={parsedColor.toHsv()}
                         pointer={() => <ColorPointer />}
-                        onChange={debounce((color) => onSelect(toColor(currentColor, { hex: toState(color).hex })))}
+                        onChange={debounce((color) => onSelect(tinycolor(color).toRgb()))}
```

Comment:
You could create an `onChangeColor` method and reuse below

Reply:
Would rather keep it like this, it's short enough and easier to understand what's going on IMO.

Reply:
okay 👍